hack/replace_serverless_chart_images.sh

```diff
@@ -16,11 +17,16 @@ if [[ ${REPLACE_SCOPE} == "main-only" ]]; then
   MAIN_ONLY_SELECTOR="| select(.version == \"main\")"
 fi
 
-IMAGES_SELECTOR=".global.images[] | select(key == \"function_*\") ${MAIN_ONLY_SELECTOR}"
-VALUES_FILE=${PROJECT_ROOT}/config/serverless/values.yaml
 
-yq -i "(${IMAGES_SELECTOR} | .directory) = \"${IMG_DIRECTORY}\"" ${VALUES_FILE}
-yq -i "(${IMAGES_SELECTOR} | .version) = \"${IMG_VERSION}\"" ${VALUES_FILE}
+if [[ ${PURPOSE} == "local" ]]; then
+  echo "Changing container registry"
+  yq -i '.global.containerRegistry.path="k3d-kyma-registry.localhost:5000"' "${VALUES_FILE}"
+fi
+
+IMAGES_SELECTOR=".global.images[] | select(key == \"function_*\") ${MAIN_ONLY_SELECTOR}"
+yq --inplace "(${IMAGES_SELECTOR} | .directory) = \"${IMG_DIRECTORY}\"" ${VALUES_FILE}
+yq --inplace "(${IMAGES_SELECTOR} | .version) = \"${IMG_VERSION}\"" ${VALUES_FILE}
 echo "==== Local Changes ===="
-yq '.global.images' ${VALUES_FILE}
+yq '.global.images' "${VALUES_FILE}"
+yq '.global.containerRegistry' "${VALUES_FILE}"
```

Comment:
I would suggest to install `yq` in `bin` if it doesn't exist. We don't need to generate another requirement for every developer/user